srcd/superset/assets/src/customization/srcdColors.jsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable sort-keys */
+
+import { CategoricalScheme } from '@superset-ui/color';
+
+const schemes = [
+  {
+    id: 'srcdMain',
+    label: 'source{d} main colors',
+    colors: [
+      '#8719cb', // royal
```

Comment:
doesn't really matter but you repeat this value with the same comment many times. Maybe it makes sense:
```js
const royal = '#8719cb';
const deepRoyal = '#400d9a';
...

colors: [royal, deepRoyal]
```

Reply:
thanks for the suggestion.
done at https://github.com/src-d/sourced-ui/pull/241/commits/3c4c93c7ecd17a877fe50baeca049c2b72b5a811#diff-71a1067ad119d25be31415985b893932